lib/bundles/inspec-compliance/api.rb

```diff
@@ -22,16 +22,14 @@ def self.profiles(config)
         profiles = JSON.parse(data)
         # iterate over profiles
         if config['server_type'] == 'automate'
-          mapped_profiles = profiles.map do |owner, ps|
-            { org: ps['owner_id'], name: owner }
-          end.flatten
+          mapped_profiles = profiles.values.to_a.flatten
         else
-          mapped_profiles = profiles.map do |owner, ps|
-            ps.keys.map do |name|
-              { org: owner, name: name }
-            end
-          end.flatten
+          mapped_profiles = []
```

Comment:
We could use another Rubyism I love... `#each_with_object`:

```
mapped_profiles = profiles.values.each_with_object([]) do |org, memo|
  memo << org.values
end
```

Again, not a blocker, just passing along something I always try and address when I see an empty object being created just to populate it with enumeration.